src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -35,9 +35,10 @@ internal class InteractiveRequest : RequestBase
             IWebUI webUi)
             : base(serviceBundle, authenticationRequestParameters, interactiveParameters)
         {
+            _webUi = webUi; // can be null just to generate the authroization uri 
```

Comment:
>can be null just to generate the authroization uri  [](start = 31, length = 51)

nit: I don't believe I will understand this sentence when not seen with this PR. how can something be null to generate something else?

Reply:
Sorry, will update the comment. I meant "can be null to support the GenerateAuthroizationUrl public method"